include/z64save.h

```diff
@@ -60,6 +60,456 @@ typedef struct {
     /* 0x24 */ s32 tempCollectFlags;
 } FaroresWindData; // size = 0x28
 
+
+// eventChkInf
+
+#define EVENTCHKINF_02 0x02
```

Comment:
I suggest putting all this after the structs/enums in this file, so it's not too broken up and a bit easier to navigate (similarly to how we leave the enums at the end of `z64actor.h` for instance)